modin/core/dataframe/algebra/groupby.py

```diff
@@ -34,12 +56,9 @@ def register(cls, map_func, reduce_func=None, **call_kwds):
 
         Parameters
         ----------
-        map_func : str, dict or callable(pandas.DataFrameGroupBy) -> pandas.DataFrame
-            If `str` this parameter will be treated as a function name to register,
-            so `map_func` and `reduce_func` will be grabbed from `groupby_reduce_functions`.
-            If dict or callable then this will be treated as a function to apply to the `GroupByObject`
-            at the map phase.
-        reduce_func : str, dict or callable(pandas.DataFrameGroupBy) -> pandas.DataFrame, optional
+        map_func : str, dict or callable(pandas.core.groupby.DataFrameGroupBy) -> pandas.DataFrame
+            Function to apply to the `GroupByObject` at the map phase.
```

Comment:
why explanation of what a `str` means is gone? if that meaning is also gone (i.e. one cannot pass a string name), then this function should not accept `str`, otherwise I don't see a point of removing the explanation

Reply:
`str` argument now has a new meaning so to say, just forgot to add it to the docstring.

Now the `str` description is in place. Appended the following to the docstring:
"If ``str`` was passed it will be treated as a DataFrameGroupBy's method name."